onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -204,10 +208,33 @@ void SaveAndCompareModels(const std::string& onnx_file, const std::basic_string<
   CompareGraphAndSessionState(session_object, session_object2);
 }
 
+/*
+static void DumpOrtModelAsJson(const std::string& model_uri) {
+  std::string ort_repo_root("path to your ORT repo root");
+  std::string ort_flatbuffers_dir(ort_repo_root + "onnxruntime/core/flatbuffers/");
+  std::string schemafile(ort_flatbuffers_dir + "ort.fbs");
+  std::string jsonfile;
+
+  ORT_ENFORCE(flatbuffers::LoadFile(schemafile.c_str(), false, &schemafile));
+  flatbuffers::Parser parser;
+  const char* include_directories[] = {ort_flatbuffers_dir.c_str(), nullptr};
+
+  ORT_ENFORCE(parser.Parse(schemafile.c_str(), include_directories));
+
+  std::string flatbuffer;
+  std::string json;
+  flatbuffers::LoadFile(model_uri.c_str(), true, &flatbuffer);
+  flatbuffers::GenerateText(parser, flatbuffer.data(), &json);
+  std::ofstream(model_uri + ".json") << json;
+}
+*/
+
 TEST(OrtModelOnlyTests, SerializeToOrtFormat) {
   const std::basic_string<ORTCHAR_T> ort_file = ORT_TSTR("ort_github_issue_4031.onnx.ort");
   SaveAndCompareModels("testdata/ort_github_issue_4031.onnx", ort_file);
 
+  // DumpOrtModelAsJson(ToMBString(ort_file));
+
```

Comment:
Do you want to add some comments why we want to dump the model in json format? Since it could be quite easy to do it outside of ORT using flatc directly.
Also maybe some comments on the helper function, such as the process maybe very slow for large models?